lib/manager/composer/__snapshots__/artifacts.spec.ts.snap

```diff
@@ -73,7 +73,7 @@ Array [
 exports[`.updateArtifacts() returns null if unchanged 1`] = `
 Array [
   Object {
-    "cmd": "composer update --with-dependencies --ignore-platform-reqs --no-ansi --no-interaction",
+    "cmd": "composer update foo bar --with-dependencies --ignore-platform-reqs --no-ansi --no-interaction",
```

Comment:
🤔 strange change

Reply:
I had to add depNames to a composer test  because of the codecoverage